### ISSUE TITLE
Unnecessary "revoke all" command before grants for tables

### ISSUE THREAD

**Issue by lopatkov** [pgadmin-org/pgadmin4#5926]
**Describe the bug**

On the SQL tab where PgAdmin generates SQL query for table creation and priviledges assignment. If we modify priviledges this SQL tab shows incorrect info. If we have assigned some priviledges for table (or all tables) it generates unnecessary "revoke all" command. Only for one of users. And if we revoke permissions for that user who has got revoke all "bonus" another one will with _revoke all_ before reallly required grant command.
Problem exists at least for tables (sequences looks like this error free).

**To Reproduce**

For example we can remove default public role and assign priviledges for DB owner and a couple of other users.

```
REVOKE USAGE ON SCHEMA public FROM PUBLIC;

GRANT ALL ON SCHEMA public TO user0;
GRANT USAGE ON SCHEMA public TO user1, user2;

GRANT ALL ON ALL TABLES IN SCHEMA public TO user0;
GRANT SELECT ON ALL TABLES IN SCHEMA public TO user1, user2;
```

**Expected behavior**

Right way - to see there granted priviledges after create table and some alter instructions.

```
GRANT ALL ON TABLE public.t1 TO user0;
GRANT SELECT ON TABLE public.t1 TO user1;
GRANT SELECT ON TABLE public.t1 TO user2;
```

**Error message**

Against expectations SQL tab contains priviledges with extra _revoke all_ query (Only for one of roles! And this role with _revoke all_ will be another one if we'll remove priviledges from user2)

```
REVOKE ALL ON TABLE public.t1 FROM user2;
GRANT ALL ON TABLE public.t1 TO user0;
GRANT SELECT ON TABLE public.t1 TO user1;
GRANT SELECT ON TABLE public.t1 TO user2;
```
At the same time \dn+ and \dp commands into psql show correct infornation.

Also at the bottom of _Propertires_ tab for table we see section "Security". Information there is equal to \dp output. So, by design there is no necessity to revoke all priviledges for role if it were not granted earlier. And this line with revoke all is confusing user.

**Screenshots**

Screenshots with info about software version and how bug is visible on SQL tab of PgAdmin. The last one with `REVOKE ALL ON TABLE public.t1 FROM user1;` was made when priviledges for user2 where removed. After that PgAdmin decided that priviledges for user1 should be revoked before grant.

![pgadmin_bug0](https://user-images.githubusercontent.com/13884420/223880220-515ea74f-61a2-4d24-926c-ba2bf5752889.png)

![pgadmin_bug1](https://user-images.githubusercontent.com/13884420/223880223-efbc83b9-9b3b-4ff0-b151-3bb3794f4557.png)

![pgadmin_bug2](https://user-images.githubusercontent.com/13884420/223880224-570fa5bf-1dd7-488b-924f-ddb0d9366cfd.png)

![pgadmin_bug3_1](https://user-images.githubusercontent.com/13884420/223880227-007ce64a-419e-49ae-a61c-1845ba938756.png)

![pgadmin_bug3_2](https://user-images.githubusercontent.com/13884420/223880228-aea34186-cdfe-4331-a683-8facb5e112bc.png)

![pgadmin_bug4](https://user-images.githubusercontent.com/13884420/223880236-cfa4fda5-b5fd-42ce-b2d5-0a43aa0b143a.png)

![pgadmin_bug5](https://user-images.githubusercontent.com/13884420/223880237-7e2177f0-1151-4196-9bac-299fbf693caa.png)

![pgadmin_bug6_1](https://user-images.githubusercontent.com/13884420/223880239-bec5fefa-8afa-4110-9752-4e0b7d572284.png)

![pgadmin_bug6_2](https://user-images.githubusercontent.com/13884420/223880241-966328e7-1686-4556-91cd-9df0e104c793.png)

![pgadmin_bug7](https://user-images.githubusercontent.com/13884420/223880215-8931164c-7672-424a-a5b8-256214118999.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 LTSC and Ubuntu
 - Version: 6.20
 - Mode: Desktop
 - Browser n/a
 - PostgreSQL: 11.18 and 15.2
 - Package type: .exe, .deb

**Additional context**

Error was found at 11version of PostgreSQL (Ubuntu) and Windows version of PgAdmin (the latest 6.20). Also it was reproduced with 15.2 version of PostgreSQL and linux (Ubuntu) version of PgAdmin (6.20 too).

**Comment by pravesh-sharma** [comment]
We have fixed a part of this issue which is `REVOKE ALL` shown for only one role which should be for all roles for which any privileges have be removed. We are showing `REVOKE ALL` to fix the issue #2844

**Comment by yogeshmahajan-1903** [comment]
As per above comment we need revoke all statement so that new table created from the sql shown sql panel is created with correct permission.
However, there was bug related to revoke all. Revoke all statement was only when for 1 user though multiple revoke commands for multiple users was required.
This issue is now fixed.

```
REVOKE ALL ON TABLE public.company1 FROM PUBLIC;
REVOKE ALL ON TABLE public.company1 FROM test role62151;
REVOKE ALL ON TABLE public.company1 FROM y.m;

GRANT INSERT ON TABLE public.company1 TO PUBLIC;

GRANT ALL ON TABLE public.company1 TO postgres;

GRANT SELECT ON TABLE public.company1 TO "test role62151";

GRANT UPDATE, DELETE ON TABLE public.company1 TO "y.m";


```
